index.html

```diff
@@ -1,67 +1,219 @@
+                </div>
+    
+                <div class="works2">
+                    <div class="image-back">
+                        <div class="professional"><h2 class="professional">Profesional Art Printing Data</h2></div>
+                        <div class="paragraphworks"><p class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works3">
+                    <div class="image-back">
+                        <div class="professional"><h2 class="professional">Profesional Art Printing Data</h2></div>
+                        <div class="paragraphworks"><p class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
+    
+                <div class="works4">
+                    <div class="image-back">
+                        <div class="professional"><h2 class="professional">Profesional Art Printing Data</h2></div>
+                        <div class="paragraphworks"><p class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
```

Comment:
- [ ] your design is currently not aligned to the left like the figma template please check to see that you are not adding margins to the technologies category and fix this design flaw :+1: 
- figma design
![fna](https://user-images.githubusercontent.com/89055502/235335196-0e41bd67-5b86-4334-85b2-6d3a0475b795.png)
- your design
![s-n-a](https://user-images.githubusercontent.com/89055502/235335202-cc1ddb8d-f3b7-49b3-92a5-1d6f2f10fe8d.png)